### PR TITLE
Enh

### LINKED ISSUE #1: new issue

### PR DESCRIPTION
**Pull request by saipradeep07**
request for adding the code snippet and resolving the issue

### CODE REVIEW

File: .ipynb_checkpoints/nonstandardcode-checkpoint.py
Comment (Karthikeyaan-Kan) [inline .ipynb_checkpoints/nonstandardcode-checkpoint.py:1]: Move the .py file to main folder

File: README.md
Comment (Karthikeyaan-Kan) [inline README.md:1]: Add more details in the script execution section

**Review by Karthikeyaan-Kan — COMMENTED** [review]
Make the necessary changes as given in the comments

File: README.md
Comment (saipradeep07) [reply README.md:1]: > Make the necessary changes as given in the comments

done

Comment (saipradeep07) [reply README.md:1]: please check

**Review by Karthikeyaan-Kan — APPROVED** [review]
The PR is approved

**Pull request merged**